Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyVersion("2.2.0.0")]
```

Comment:
Do we need to go out to 4 digits for these? We've typically only used 3 for MRTK.

Reply:
the file format uses 4.

Reply:
i had visual studio create one and used the same format